internal/goldendataset/metric_gen.go

```diff
@@ -0,0 +1,201 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goldendataset
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/internal/data"
+)
+
+// Simple utilities for generating metrics for testing
+
+type MetricCfg struct {
+	MetricDescriptorType pdata.MetricType
+	NumBuckets           int
+	NumIlm               int
+	NumMetrics           int
```

Comment:
I think this is NumMetricsPerIl

Reply:
Sorry, it turns out this is not used after a recent change. Removed.